tests/pkg_relic/Makefile

```diff
@@ -28,7 +28,7 @@ BOARD_INSUFFICIENT_MEMORY :=    nucleo-f030r8 \
                                 nucleo-l053r8 \
                                 #
 
-export TEST_BOARDS_LLVM_COMPILE := native
+TOOLCHAIN_BLACKLIST += llvm
```

Comment:
Indeed it is the same issue, so you can reuse the same comment as `jerryscript`.

It compiles with `llvm` with this fix, I can do a PR for the fix:

``` diff
diff --git a/pkg/relic/Makefile b/pkg/relic/Makefile
index 7e2466c9d..d73aecbee 100644
--- a/pkg/relic/Makefile
+++ b/pkg/relic/Makefile
@@ -18,7 +18,7 @@ all: $(PKG_BUILDDIR)/Makefile

 $(PKG_BUILDDIR)/Makefile: $(TOOLCHAIN_FILE)
        cd $(PKG_BUILDDIR) && \
-       COMP="$(filter-out -Werror -Werror=old-style-definition -Werror=strict-prototypes -std=gnu99, $(CFLAGS) ) " \
+       COMP="$(INCLUDES) $(filter-out -Werror -Werror=old-style-definition -Werror=strict-prototypes -std=gnu99, $(CFLAGS) ) " \
        cmake -DCMAKE_TOOLCHAIN_FILE=$(TOOLCHAIN_FILE) \
                  -DCHECK=off -DTESTS=0 -DBENCH=0 -DSHLIB=off -Wno-dev $(RELIC_CONFIG_FLAGS) .

```

Reply:
so just to clarify: its either blacklisting llvm for the relic package **or** applying your fix for jerryscript to make it work with llvm? Or is both somehow needed?

Reply:
I would say it is now blacklisting because it does not work for `llvm` and `cortex-m`, it is a fact, and it is an immediate solution.

Making it work for `llvm` is an improvement that needs work, so a new feature somehow with specific review. So can come in a second step.

I do not delay this PR because of something that was broken for years.
I was just adding details so the `blacklisting` comment would be accurate.